gammapy/scripts/tests/test_spectrum_pipe.py

```diff
@@ -0,0 +1,13 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from numpy.testing import assert_equal, assert_almost_equal
+from astropy.io import fits
+from astropy.tests.helper import remote_data
+from gammapy.scripts import GammapySpectrumAnalysis
+from ...datasets import get_path
+import yaml
+
+def test_spectrum_pipe(tmpdir):
+
+    #Change to remote file in the end
+    configfile = "~/Software/gammapy/gammapy/scripts/spectrum_pipe_example.yaml"
```

Comment:
This test currently fails:
https://travis-ci.org/gammapy/gammapy/jobs/82537347#L1483

Either mark it `@pytest.mark.xfail` and add a `TODO: implement test later`.
Or actually implement the test using this one run for now:
https://github.com/gammapy/gammapy-extra/tree/master/test_datasets/irf/hess/pa